portfolio-project-matching-app/models/Project.js

```diff
@@ -14,6 +14,22 @@ class Project {
         this.users = null;
     }
 
+    hasTechnology(technologyId) {
```

Comment:
Tried my hand at a class method that seemed appropriate. It is purely additive, hope I didn't step too far out of bounds here.